### PR TITLE
Traefik labels

### PR DESCRIPTION
**Pull request by dajuno**
# MaRDI Pull Request

**Changes**:
- replace traefik fileProvider configurations by docker labels
- fixes issue of hardcoded, but varying, service names

**Instructions for PR review**:
- [ ] Conceptual Review (Logic etc...) 
- [ ] Code Review (Review your implementation) 
- [ ] Checkout (Test changes locally) 

**Checklist for this PR**: 
- [ ] [Reviewers and Assignee specified.](https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users)
- [ ] [All related issues are linked.](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)

### CODE REVIEW

**Comment by physikerwelt** [comment]
image
I suggest adding reviewers after PRs are ready for review

File: docker-compose.yml [hunk 1]
```diff
@@ -45,6 +45,10 @@ services:
       WIKIBASE_PINGBACK:
       MW_WG_ENABLE_UPLOADS:
       <<: *wikibase_extra_variables
+    labels:
+      - traefik.http.routers.service-wikibase.rule=Host(`mardi.formulasearchengine.com`,`wikibase.mardi.schubotz.org`,`mardi.schubotz.org`,`portal.mardi4nfdi.de`)
```
Comment (physikerwelt) [inline docker-compose.yml:49]: You can drop the schubotz.org and formulasearchengine domain aliases.

**Review by physikerwelt — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (dajuno) [reply docker-compose.yml]: done. what about the formulasearchengine network aliases for the quickstatements and wdqs-frontend containers?

Comment (physikerwelt) [reply docker-compose.yml]: This should be removed as well. Thank you

File: docker-compose.yml [hunk 2]
```diff
@@ -231,7 +231,7 @@ services:
         aliases:
          - quickstatements.formulasearchengine.com
```
Comment (physikerwelt) [inline docker-compose.yml:232]: this one needs to be changed.

File: docker-compose.yml [hunk 3]
```diff
@@ -167,7 +167,7 @@ services:
       - WIKIBASE_HOST=wikibase.svc
```
Comment (physikerwelt) [inline docker-compose.yml:167]: - wdqs.formulasearchengine.com needs to go away as well.

File: docker-compose.yml (hunk 2 above)
Comment (physikerwelt) [reply docker-compose.yml]: I figured out that simple edits can be performed in the editor directly. This is more efficient than writing comments and requesting the change

**Review by physikerwelt — APPROVED** [review]

**Pull request merged**